src/io/flutter/project/FlutterProjectOpenProcessor.java

```diff
@@ -102,4 +114,33 @@ private ProjectOpenProcessor getDelegateImportProvider(@Nullable VirtualFile fil
   public boolean isStrongProjectInfoHolder() {
     return true;
   }
+
+  private static class PackagesOutOfDateNotification extends Notification {
+
+    @NotNull
+    private final Project myProject;
+
+    public PackagesOutOfDateNotification(@NotNull Project project) {
+      super("Flutter Packages", FlutterIcons.Flutter, "Package updates.",
+            null, "This project's packages are ready to" +
```

Comment:
Isn't there a pretty big difference between 'get' and 'upgrade'? And doesn't missing packages just imply 'get' is needed', not 'upgrade'?